src/tax-lot/tax-lot.repository.ts

```diff
@@ -17,7 +17,14 @@ import {
   FindZoningDistrictsByBblRepo,
   FindZoningDistrictClassesByBblRepo,
   FindManyRepo,
+  FindManyBySpatialFilterRepo,
+  FindGeomBufferRepo,
+  FindMaximumInscribedCircleCenterRepo,
+  CheckGeomIsValidRepo,
+  FindGeomFromGeoJsonRepo,
 } from "./tax-lot.repository.schema";
+import { Geometry } from "geojson";
+import { Geom } from "src/types";
 
 export class TaxLotRepository {
```

Comment:
A lot of these geospatial methods strike me as things that can be reused, possibly into some sort of general purpose Repository class for geospatial entities. I'm cool with putting that off until we have more than just the Tax Lot repo that need them, but wanted to make a note of it.